src/Bridges/Constraint/zero_one.jl

```diff
@@ -0,0 +1,96 @@
+"""
+    ZeroOneBridge{T}
+
+The `ZeroOneBridge` splits a `MOI.SingleVariable`-in-`MOI.ZeroOne` constraint
+into a `MOI.SingleVariable`-in-`MOI.Integer` constraint
+and a `MOI.SingleVariable`-in-`MOI.Interval(0,1)` constraint.
+"""
+struct ZeroOneBridge{T} <: AbstractBridge
+    variable_index::MOI.VariableIndex
```

Comment:
You don't need to store this. In fact, the bridge could work for any set (not just `MOI.SingleVariable`) )without much modification and you don't want to store the function.